testdata/baselines/reference/submodule/compiler/callsOnComplexSignatures.js.diff

```diff
@@ -38,21 +38,19 @@
      // Pair of non-like intrinsics
      function render(url) {
 -        var Tag = url ? 'a' : 'button';
```

Comment:
I'm working on dropping ES5 out of the submodule, so theoretically that'd eliminate a load of diffs here too.